RecoLocalTracker/SiStripClusterizer/plugins/SiStripClusters2ApproxClusters.cc

```diff
@@ -20,6 +20,10 @@
 #include "FWCore/ParameterSet/interface/FileInPath.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "DataFormats/SiStripCluster/interface/SiStripApproximateCluster.h"
+#include "DataFormats/SiStripCluster/interface/SiStripApproximateClusterCollection.h"
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
```

Comment:
```suggestion
#include "DataFormats/SiStripCluster/interface/SiStripApproximateClusterCollection.h"
```
Three includes are duplicated of the already included ones.
And please move up together the other DataFormats includes

Reply:
By the way, since only DetSetVectorNew seems to be used, I think you can remove the includes of "DataFormats/Common/interface/DetSetVector.h"